docs/contributing/update_ami_versions.md

```diff
@@ -0,0 +1,16 @@
+# Updating The Default Base AMI
+
+With the release of `kOps 1.18`, the base AMI was switched over from a pre-baked `kOps` AMI to the official Ubuntu 20.04 LTS image.
+This makes the image update process easier, because kOps contributors no longer have to fully build an test a full AMI, but rather base off of the latest stable Ubuntu image.
```

Comment:
```suggestion
This makes the image update process easier, because kOps contributors no longer have to fully build and test a full AMI, but rather base off of the latest stable Ubuntu image.
```